elastic/logstash
# Added plugin.name to fish tag log lines

Added the plugin's name to log lines generated by the plugins

----

This PR adds `plugin.id` to log lines.

Tipical use case:
- the user sees some log lines with `plugin.id` bring that id (running **Java pipeline**)
- query the LS Node Info API to get source reference in config file for that id (`curl -XGET "localhost:9600/_node/pipelines/main?graph=true"`)

**How to test**
- run logstash with java pipeline, with debug log running the following pipeline.conf:
```
input {
  stdin {}
}

filter {
  json {
    source => "message"
  }
  sleep {
    id => "sleep_filter_123"
    time => 1
  }
}

output {
  stdout {
    codec => rubydebug
  }
}
```
`bin/logstash -f "<config>" --java-execution=true  --debug`
- in console or in `logs/logstash-plain.log` you should find a line like:
```
[DEBUG][logstash.filters.json    ][main][31543d02478e75487017793958c743af64669fe5f915ab05695717240d026f49] Running json filter {:event=>#<LogStash::Event:0x2220404c>}
```
- bring that uuid and search in Node Into API with a rest call
